static/js/modules/tables.js

```diff
@@ -589,8 +589,9 @@ DataTable.prototype.fetch = function(data, callback) {
 
 DataTable.prototype.export = function() {
   var url = this.buildUrl(this.api.ajax.params(), false);
-  download.download(url);
+  var item = download.download(url);
   this.disableExport({message: DOWNLOAD_MESSAGES.pending});
+  item.$body.find('.button').focus();
```

Comment:
If possible, prefer not to have one component (the table) reach down into a different component (the download) to call methods on its children--this requires a developer working on the table component to know a lot about the internals of the download component. What if `DownloadItem.init` handled the focus instead?